cluster/loadbalance/consistent_hash.go

```diff
@@ -85,22 +85,25 @@ func (lb *ConsistentHashLoadBalance) Select(invokers []protocol.Invoker, invocat
 	return selector.Select(invocation)
 }
 
-// Uint32Slice ...
+// Uint32Slice Slice of uint32
 type Uint32Slice []uint32
```

Comment:
I think make this struct to private would be better.

Reply:
please move this to gost. We already have similar structure...

Reply:
> please move this to gost. We already have similar structure...

+1

Reply:
already replaced Uint32Slice  by gxsort.Uint32Slice